msvc/unity.props

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <PLATFORM_UNITY_DEFINES>PLATFORM_UNITY</PLATFORM_UNITY_DEFINES>
+    <PLATFORM_UNITY_INCLUDE_DIR>../../../il2cpp/build/libil2cpp/os/c-api</PLATFORM_UNITY_INCLUDE_DIR>
```

Comment:
I think we need to figure out a solution to this circular dependency..maybe move the il2cpp platform abstraction layer into github?